cosmolog/cosmologger.py

```diff
@@ -258,22 +258,22 @@ def __init__(self, *args, **kwargs):
         self._version = kwargs.pop('version')
         logging.Formatter.__init__(self, *args, **kwargs)
 
-    def _prepare_exception(self, record):
-        return record.exc_text
-
     def _prepare_payload(self, record):
-        return record.__dict__.get('payload', {})
+        pld = record.__dict__.get('payload', {})
+        if record.exc_info:
+            pld['exc_text'] = record.exc_text
```

Comment:
should we add this to the readme as a reserved payload key?

Reply:
We should probably say something. Here are the official docs on `exc_text`: https://docs.python.org/2/library/logging.html#logging.Formatter.format